src/main/java/ca/serum390/godzilla/api/filters/ReactRouterForwardFilter.java

```diff
@@ -29,11 +28,8 @@
      */
     private boolean matchesReactRouterRoute(ServerWebExchange exchange) {
         String path = exchange.getRequest().getURI().getPath();
-        return !path.startsWith("/api/")
-            && !path.startsWith("/resources/")
-            && !path.endsWith(".js")
-            && !path.endsWith(".css")
-            && !path.endsWith(".svg");
+        return !path.startsWith("/api/") && !path.startsWith("/goods/") && !path.startsWith("/resources/")
```

Comment:
We are adding REST APIs to the `/api/` parent route, so there is no need to modify the filter.

Can you undo this change? And reformat so that this expression is readable:

```java
        return !path.startsWith("/api/")
            && !path.startsWith("/resources/")
            && !path.endsWith(".js")
            && !path.endsWith(".css")
            && !path.endsWith(".svg");
```